### PR TITLE
fixing pangu bugs, device in timeloop and missing channels in scale

### PR DESCRIPTION
**Pull request by yairchn**
# Earth-2 MIP Pull Request
## Description
two bugs appeared in running pangu:
1. device is missing in time loop 
2. the channel `q1000` in pangu does not have a scale. 

This PR handles these two problem, related to #70 

## Checklist

- [x] I am familiar with the Contributing Guidelines.
- [x] New or existing tests cover these changes.
- [x] The documentation is up to date with these changes.
- [x] The CHANGELOG.md is up to date with these changes.
- [x] An [issue](https://github.com/NVIDIA/earth2mip/issues) is linked to this pull request.

### CODE REVIEW

File: earth2mip/inference_ensemble.py [hunk 1]
```diff
@@ -252,10 +252,13 @@ def perturb(x, rank, batch_id, device):
         if rank == 0 and batch_id == 0:  # first ens-member is deterministic
             noise[0, :, :, :, :] = 0
 
-        scale = torch.tensor(
-            [channel_stds[channel] for channel in model.in_channel_names],
-            device=x.device,
-        )
+        scale = []
```
Comment (NickGeneva) [inline earth2mip/inference_ensemble.py:255]: Please add a comment here discussing what this is doing e.g.:

`# for channels that are not present in normalization dictionary, perturbation will be skipped by default`

For those looking in the code some more

File: earth2mip/networks/pangu.py [hunk 2]
```diff
@@ -194,6 +194,10 @@ def grid(self):
     def n_history(self):
         return 0
 
+    @property
+    def device(self) -> torch.device:
+        return "cuda"
```
Comment (NickGeneva) [inline earth2mip/networks/pangu.py:199]: Sorry, I know I told you this, but we can make this just a little more robust with
```
from modulus.distributed.manager import DistributedManager
```


```
@property
    def device(self) -> torch.device:
        return DistributedManager().device
```

File: examples/notebooks/02_model_comparison.ipynb [hunk 3]
```diff
@@ -165,9 +188,22 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 7,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
```
Comment (NickGeneva) [inline examples/notebooks/02_model_comparison.ipynb:193]: You need to clean your notebook.

I would suggest doing:
`git checkout 00876b7 -- examples/notebooks/02_model_comparison.ipynb`

Then remake your edits without running the notebook, and then commit.

**Review by NickGeneva — CHANGES_REQUESTED** [review]
